### DISCUSSION TITLE
[Informationssystem] Klassenbuch Lehrer

### DISCUSSION THREAD

**Discussion by tilwbr** [SE-TINF22B2/G2-Schoolify#27] [User Stories]
### User Story
 Als Lehrer möchte ich Einträge ins Klassenbuch über den Schulplaner machen können, damit kein lästiges Buch per Hand rumgereicht wird und kein Buch verloren gehen kann
### Beschreibung:
Es sollte eine eigene Seite geben für "digitale Klassenbücher", wo Lehrer zwischen den Klassen auswählen können, um dann für die jeweilige Klasse einen Klassenbucheintrag für einen bestimmten Tag zu erstellen. Zudem soll einen alter Tag ausgewählt werden können, um alte Einträge anzeigen zu lassen.

**Optional:**
- Filterfunktionen

### Akzeptanz-Kriterien:
- [ ] Auswahl zwischen den Klassen eines Lehrers
- [ ] Alte Klassenbucheinträge sollen ausgewählt werden können 
- [ ] Neue Einträge sollen für einen bestimmten Tag erstellt werden können

### Issues
- #119 
- #120

**Comment by two-Gee** [comment]
passt nicht zu einer Schulwebseite

**Reply by Kronprinz03** [reply]
Finde ich schon so könnten Notizen des Lehrers, die für die Schüler/Eltern relevant sind direkt online angezeigt werden. Man müsste aber glaube ich differenzieren
